src/test/java/com/gamegoo/gamegoo_v2/integration/auth/AuthFacadeServiceTest.java

```diff
@@ -72,13 +77,15 @@ void login() {
             LoginResponse response = authFacadeService.login(loginRequest);
 
             // then
+            // response 검증
             assertThat(response).isNotNull();
             assertThat(response.getAccessToken()).isEqualTo(jwtProvider.createAccessToken(member.getId()));
             assertThat(response.getRefreshToken()).isEqualTo(jwtProvider.createRefreshToken(member.getId()));
             assertThat(response.getId()).isEqualTo(member.getId());
             assertThat(response.getName()).isEqualTo(member.getGameName());
             assertThat(response.getProfileImage()).isEqualTo(member.getProfileImage());
 
+            refreshTokenRepository.findByMember(member).ifPresent(refreshToken -> assertThat(refreshToken.getRefreshToken()).isEqualTo(response.getRefreshToken()));
```

Comment:
이 부분은 refreshTokenRepository에 해당 member를 갖는 데이터가 있을 때에만 assert를 하고 있어서 아래와 같이 변경하는게 좋을 것 같습니다..! 기존 방식은 refreshTokenRepository에 데이터가 저장이 안됐을 때에도 테스트를 통과합니다
```suggestion
            RefreshToken refreshToken = refreshTokenRepository.findByMember(member).orElseThrow();
            assertThat(refreshToken.getRefreshToken()).isEqualTo(response.getRefreshToken());
```

Reply:
헉 넵 알겠습니다!